.coveralls.yml

```diff
@@ -0,0 +1,3 @@
+repo_token: Fc9rievh0XnhuLbMs3cMqi7rDcvQhLzLw
```

Comment:
This needs to be removed and re-generated. Then it should be added to travis as a environment variable `COVERALLS_REPO_TOKEN`. It is not meant to be public.